src/transaction/log_checkpoint_info.cpp

```diff
@@ -0,0 +1,159 @@
+
+#include "log_checkpoint_info.hpp"
+namespace cublog
```

Comment:
Leave an empty space after includes.
```suggestion
#include "log_checkpoint_info.hpp"

namespace cublog
```